js/src/elements/form-behavior.ts

```diff
@@ -356,10 +358,20 @@ export class FormBehaviorElement extends HTMLElement {
 
 	validateRequired(input: HTMLValueElement, label: string): string | null {
 		if (input.validity.valueMissing) {
-			return `${this.locStrings.inputRequired.replace(
+			return this.locStrings.inputRequired.replace(
 				'{inputLabel}',
 				customElements.get(input.localName) ? `A selection for "${label}"` : label
-			)}`;
+			);
+		}
+		return null;
+	}
+
+	validateGroupRequired(input: HTMLValueElement, label: string): string | null {
+		const group = getField(input);
+		const inputs = Array.from(group.querySelectorAll<HTMLInputElement>('input'));
```

Comment:
Do we want to check _all_ types inputs are checked? or only certain types? Should we query `input[type="checkbox"],input[type="radio"]`?

Reply:
removed method.  No longer needed.